addons/hr_holidays/tests/test_leave_requests.py

```diff
@@ -197,10 +197,10 @@ def test_allocation_request(self):
         """ Create an allocation request """
         # employee should be set to current user
         allocation_form = Form(self.env['hr.leave.allocation'].with_user(self.user_employee))
-        allocation_form.name = 'New Allocation Request'
         allocation_form.holiday_status_id = self.holidays_type_2
         allocation_form.date_from = date(2019, 5, 6)
         allocation_form.date_to = date(2019, 5, 6)
+        allocation_form.name = 'New Allocation Request'
```

Comment:
useless diff?

Reply:
Nopes, so here, this is because of the following change:

```
# Useless depends, so that name is computed on new, before saving the record
    @api.depends_context('uid')
    @api.depends('holiday_status_id')
    def _compute_description(self):
```
So, basically, previously, allocation name was not computed on create. Now, it's not a case anylonger.

With this change, this test was failing as first the name field was assigned, then holiday_status_id;
Which caused for the name field to be recomputed and assigned to False. As the field is required, leave_form.save() was raising an error.